pkg/kubelet/pod_workers.go

```diff
@@ -291,9 +291,9 @@ func killPodNow(podWorkers PodWorkers, recorder record.EventRecorder) eviction.K
 		}
 
 		// we timeout and return an error if we don't get a callback within a reasonable time.
-		// the default timeout is relative to the grace period (we settle on 2s to wait for kubelet->runtime traffic to complete in sigkill)
+		// the default timeout is relative to the grace period (we settle on 10s to wait for kubelet->runtime traffic to complete in sigkill)
 		timeout := int64(gracePeriod + (gracePeriod / 2))
-		minTimeout := int64(2)
+		minTimeout := int64(10)
```

Comment:
I initially proposed 4s because in 30 evictions, the longest time I recorded was 3.2 seconds.  However, this is likely still a little low, because 30 runs isn't a huge sample size.  With 1000 runs, the max may very well be in the 5-10 second range, or at least likely higher than 4s.  AFAIK we don't have any major problems with hung container deletion operations, so a higher value seems like the safer option.  I will be monitoring this, and we can always adjust it downwards if we find issues